awesome_page.md

```diff
@@ -0,0 +1,14 @@
+## markdown playground
+- **this is bold**
+- *this is italic using a single star (better)* _this is italics using an underscore_
+- this is `inline code`
+
+```ruby
+def hello
+  puts "hello, world!"
+end
+```
```

Comment:
great job with the syntax highlighting! :high_brightness: 

![screenshot_0272](https://cloud.githubusercontent.com/assets/2850369/7840549/8931589a-045d-11e5-8322-aaa6f2714591.png)